### DISCUSSION TITLE
How to avoid duplication when we have 3 prometheus instances

### DISCUSSION THREAD

**Discussion by sivamalla42** [grafana/mimir#7336] [Help and support]
Hi All,

We have not enabled the deduplication as we have only one prometheus instance in all lower environment, but in prod we have three instances. We started observing the data three times of what we are expecting. can someone please help in how to use the labels here if three prometheus instances are available?

**Comment by pstibrany** [comment]
Hello, setup where there are multiple Prometheus servers sending the same data for high-availability reasons is supported by Mimir by using HA Deduplication. See the documentation here: https://grafana.com/docs/mimir/latest/configure/configure-high-availability-deduplication/
